src/wazuh_modules/wm_database.c

```diff
@@ -565,7 +565,6 @@ int wm_sync_file(const char *dirname, const char *fname) {
         }
 
         free(name);
-        result = wm_sync_agent_group(id_agent, fname);
```

Comment:
I think that the whole `case WDB_GROUPS:` section can be removed, now that the groups files don't exist any more.

There are another calls related, like `wm_scan_directory(GROUPS_DIR);` that should be also considered for removal.

Reply:
I only removed the code that you suggested. There is more code to be deprecated but that will be done in a separate issue.
DONE.